src/voku/helper/AntiXSS.php

```diff
@@ -775,6 +775,19 @@ private function _do_never_allowed(string $str)
     private function _do_never_allowed_afterwards(string $str)
     {
         if (\stripos($str, 'on') !== false) {
```

Comment:
I dont really get how the actual regex work later on in this method.
maybe we could get a better knock-out pattern in this line already (which contains more characters)